iocore/net/UnixNetVConnection.cc

```diff
@@ -1249,17 +1238,13 @@ UnixNetVConnection::populate(Connection &con_in, Continuation *c, void *arg)
   this->thread = this_ethread();
 
   EThread *t = this_ethread();
-  if (ep.start(get_PollDescriptor(t), this, EVENTIO_READ | EVENTIO_WRITE) < 0) {
-    // EEXIST should be ok, though it should have been cleared before we got back here
-    if (errno != EEXIST) {
-      Debug("iocore_net", "populate : Failed to add to epoll list");
-      return EVENT_ERROR;
-    }
+  if (get_NetHandler(t)->startIO(this) < 0) {
```

Comment:
It seems we using NetHandler without lock.

Reply:
Yes, it is a bug. I will create another PR to resolve it.

Reply:
@scw00 I have fixed it. Try lock NetHandler first then do ```startIO```.